public/app/features/dashboard-scene/inspect/HelpWizard/utils.ts

```diff
@@ -62,15 +63,29 @@ export function getGithubMarkdown(panel: VizPanel, snapshot: string): string {
 }
 
 export async function getDebugDashboard(panel: VizPanel, rand: Randomize, timeRange: TimeRange) {
-  let saveModel;
+  let saveModel: ReturnType<typeof gridItemToPanel> = { type: '' };
   const isLibraryPanel = panel.parent instanceof LibraryVizPanel;
   const gridItem = (isLibraryPanel ? panel.parent.parent : panel.parent) as DashboardGridItem;
+  const scene = panel.getRoot() as DashboardScene;
 
   if (isLibraryPanel) {
     saveModel = {
       ...gridItemToPanel(gridItem),
       ...vizPanelToPanel(panel),
     };
+  } else if (scene.state.editPanel) {
+    // If panel edit mode is open when the user chooses the "get help" panel menu option
+    // we want the debug dashboard to include the panel with any changes that were made while
+    // in panel edit mode.
+    const sourcePanel = scene.state.editPanel.state.vizManager.state.sourcePanel.resolve();
+    const parentItem = sourcePanel.parent as DashboardGridItem | LibraryVizPanel;
```

Comment:
Nit: I think we can skip this type assertion to reduce betterer results, and just do:
`const dashGridItem = sourcePanel.parent instanceof LibraryVizPanel ? sourcePanel.parent.parent : sourcePanel.parent;`
But it's up to you.